miner/miner.go

```diff
@@ -476,6 +467,11 @@ func (m *Miner) computeTicket(ctx context.Context, brand *types.BeaconEntry, bas
 		buf.Write(base.TipSet.MinTicket().VRFProof)
 	}
 
+	worker, err := m.api.StateAccountKey(ctx, mbi.WorkerKey, types.EmptyTSK)
```

Comment:
Isn't mbi.WorkerKey already a key address?

Reply:
You're right, it is.